src/main/java/manga/Manga.java

```diff
@@ -50,6 +52,16 @@ public void deleteDeadline() {
         this.deadline = "None";
     }
 
+    //@@author sarahchow03
+
+    public Sale getSalesData() {
+        return salesData;
+    }
+
+    public void addSalesData(Sale salesData) {
```

Comment:
In aligning ourselves with the concepts of getters and setters, might we consider changing `addSalesData` to `setSalesData`? Just a minor point.